src/engine/sys/sdl_input.cpp

```diff
@@ -427,7 +427,7 @@ void IN_SetMouseMode(MouseMode newMode)
 		}
 
 		SDL_ShowCursor( newMode == MouseMode::SystemCursor ? SDL_ENABLE : SDL_DISABLE );
-		auto grab = newMode != MouseMode::Deltas || in_nograb->integer ? SDL_FALSE : SDL_TRUE;
+		auto grab = in_nograb->integer ? SDL_FALSE : SDL_TRUE;
```

Comment:
@slipher do it work for you if I remove this change?

Reply:
Yes, the bug goes away upon reverting that part.

Reply:
Great, reverting that part then.